docs/data/material/getting-started/usage/usage.md

```diff
@@ -42,17 +43,4 @@ To ensure proper rendering and touch zooming for all devices, add the responsive
 ### CssBaseline
 
 Material UI provides an optional [CssBaseline](/material-ui/react-css-baseline/) component.
-It fixes some inconsistencies across browsers and devices while providing slightly more opinionated resets to common HTML elements.
-
-## Versioned documentation
-
-This documentation always reflects the latest stable version of Material UI.
-You can find older versions of the documentation on a [separate page](https://mui.com/versions/).
-
-## Next steps
-
-Now that you have an idea of the basic setup, it's time to learn more about:
-
-- How to provide [the Material Design font and typography](/material-ui/react-typography/).
```

Comment:
If we remove the link to the font, I think that we should add a new h3 under the h2 Globals to link how to install the Material Design font: https://mui.com/material-ui/react-typography/#general.

Reply:
Ah right, my bad, it's already covered in the previous section in Installation.